docs/pages/machine-id/getting-started.mdx

```diff
@@ -0,0 +1,120 @@
+---
+title: Machine ID Getting Started Guide
+description: Getting started with Teleport Machine ID
+---
+
+# Getting Started
+
+In this getting started guide we will use Machine ID to create an identity for a machine and use that identity to connect to a SSH server.
+
+Here's an overview of what we will do:
+
+1. Download and install Teleport (=teleport.version=).
+2. Create Bot User
+3. Start Machine ID
+4. Use Machine ID
+
+## Prerequisites
+
+Before using Machine ID, you will need an existing Teleport cluster.
+
+If you have not setup a Teleport cluster before, follow the [Getting started on a Linux Server](https://goteleport.com/docs/getting-started/linux-server/) guide for instructions.
```

Comment:
Should we link to the Getting Started index page in case the user wants to try Teleport on a different environment (e.g. Kubernetes or a local docker-compose cluster) or is more interested in Cloud? https://goteleport.com/docs/getting-started/